src/memory_region/local.rs

```diff
@@ -130,7 +129,7 @@ impl LocalMr {
         } else {
             Ok(LocalMrSlice::new(
                 self,
-                self.addr().overflow_add(i.start),
+                self.addr().saturating_add(i.start),
```

Comment:
Why not `wrapping_add`?

Reply:
I think both can be used here, but why `wrapping_add` is better?

Reply:
Using `saturating_add` means that it is possible to overflow and then I want the result to be `<$ty>::MAX`.
`wrapping_add` should be used when the addition never overflows.